controllers/submariner/np_syncer_resources.go

```diff
@@ -123,3 +123,7 @@ func newNetworkPluginSyncerDeployment(cr *v1alpha1.Submariner, clusterNetwork *n
 
 	return networkPluginSyncerDeployment
 }
+
+func canInstallNetworkPluginSyncer(submariner *v1alpha1.Submariner) bool {
```

Comment:
Bit of a nitpick, but wouldn’t this be better as `needsNetworkPluginSyncer`?

Reply:
Sure